packages/shared/lib/services/proxy.service.ts

```diff
@@ -298,6 +299,32 @@ class ProxyService {
         return false;
     };
 
+    private isProviderRetryTriggered(retryConfig: RetryHeaderConfig | undefined, response: AxiosError): boolean {
+        if (!retryConfig) {
+            return false;
+        }
+
+        const { status_code, body_contains } = retryConfig;
+
+        const statusCodeValid =
+            status_code && status_code.includes('x')
```

Comment:
really not sure we need that feature, they return a 400 let's support a 400 and see if we need more

Reply:
Specifically added this because Linear's docs say:

> Once you actually exceed rate limits, Linear API will start returning rate limit error responses. You can catch these by inspecting response http status code to be 4xx (>=400 and <500) and response body containing RATELIMITED error code.

https://developers.linear.app/docs/graphql/working-with-the-graphql-api/rate-limiting